Rawl_B502/bunad_laigen.ttl

```diff
@@ -192,270 +239,383 @@
+    irishRel:nomName "Sírna" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Sirna>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Déin>
     a irishTitle:Rí ;
     irishRel:genName "Déin" ;
-    irishRel:nomName "Dian" .
+    irishRel:nomName "Dian" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Dian>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Demál>
     a foaf:Person ;
-    irishRel:nomName "Demál" .
+    irishRel:nomName "Demál" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Demáil>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Rothait>
     a foaf:Person ;
-    irishRel:nomName "Rothait" .
+    irishRel:nomName "Rothait" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Rechtaid>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Ogamuin>
     a irishTitle:Rí ;
-    irishRel:nomName "Ogamuin" .
+    irishRel:nomName "Ogamuin" ;
+    rel:ancestorOf <#NuaduNecht>.
+
+# owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Móen> ? EPT
 
 <#Óengus-902e76c6>
     a foaf:Person ;
     irishRel:nomName "Óengus" ;
-    rel:childOf <#Eochu-71d7ebad>.
-    
+    rel:childOf <#Eochu-71d7ebad>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Óengus>;
+    rel:ancestorOf <#NuaduNecht>.
+
 <#Eochu-71d7ebad>
     a foaf:Person ;
-    irishRel:nomName "Eochu" .
+    irishRel:nomName "Eochu" ;
+    rel:ancestorOf <#NuaduNecht>.
     #same as <#Eochu> above? CY
 
 <#Smirgnath>
     a foaf:Person ;
-    irishRel:nomName "Smirgnath" .
+    irishRel:nomName "Smirgnath" ;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Smrith>
     a foaf:Person ;
-    irishRel:nomName "Smrith" .
+    irishRel:nomName "Smrith" ;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Enbath>
     a foaf:Person ;
-    irishRel:nomName "Enbath" .
+    irishRel:nomName "Enbath" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Senboth>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Tigernmas>
     a foaf:Person ;
-    irishRel:nomName "Tigernmas" .
+    irishRel:nomName "Tigernmas" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Tigernmas>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Ethér>
     a foaf:Person ;
-    irishRel:nomName "Ethér" .
+    irishRel:nomName "Ethér" ;
+    rel:ancestorOf <#NuaduNecht>.
+
+# owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Ethrél> ?
 
 <#Éremón>
     a foaf:Person ;
-    irishRel:nomName "Éremón" .
+    irishRel:nomName "Éremón" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#h-Éremóin>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Mílid>
     a foaf:Person ;
     irishRel:genName "Mílid" ;
-    irishRel:nomName "Míl" .
+    irishRel:nomName "Míl" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Mílid>;
+    rel:ancestorOf <#NuaduNecht>.
+
     #same As http://example.com/LL/eoganachta_casil.ttl#MiledEspain ? CY
+    # Surely so? There's only one Míl Espaine (at least, there'd better
+    # be...) EPT
```

Comment:
Having a discussion in the comments is probably not a good idea, if you think this is the same then it is probably a good idea to just add the owl:sameAs link and delete the comments. And we don't want clones of Míl running around :P